create_graph.py

```diff
@@ -11,19 +11,33 @@ def process_pkl_files(triples_file,labels_file):
     triples_df = pd.read_csv(triples_file,sep = '	')
     triples_df.columns.str.lower()
 
-    labels = pd.read_csv(labels_file, sep = '	')
-    labels.columns.str.lower()
-
-    return triples_df,labels
+    #labels = pd.read_csv(labels_file, sep = '	')
+    #labels.columns.str.lower()
+    
+    
+    # In order to show all the labels available in the labels_file tab deliminated txt file, read the file with pandas fixed-width lines reader
+    # (this will destroy the frame of the file though!!!)
+    labels_fwf = pd.read_fwf(labels_file)
+    print("The total number of labels is", len(labels_fwf))
+    
+    '''extract the URIs from the fwf_labels file.
+    uri_labels = []
+    for i in labels_fwf.iloc[:,0]:
```

Comment:
Can we just treat this as a df and pass as all_labels? In the later subgraph functions (for example prioritize_path_cs which uses find_all_shortest_paths) we use the label (not uri identifier) since we expect the input file to be in the form of labels, not Uris. I think we need to pass all columns no just a list of the uris